luisa_cortezano/ejercicio2.md

```diff
@@ -0,0 +1,22 @@
+```
+Sub datos()
+    f = 1
+    sw = True
+    
+    c = Int(InputBox("ingrese numero de cedula"))
+    
+    While sw
```

Comment:
The limit variable must be in true so that you can end the loop. If not, the code will run infinitely.